posts/2022-06-16-sparse-registry-testing.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Cargo sparse-registry call for testing"
+author: Arlo Siemsen
+team: The Cargo Team <https://www.rust-lang.org/governance/teams/dev-tools#cargo>
+---
+
+The Cargo nightly [`sparse-registry`][sparse-registry] feature is ready for testing. The
+feature causes Cargo to access the crates.io index over HTTP, rather than git. It can
+provide a significant performance improvement, especially if the git index is
+out-of-date or not yet cloned.
+
+## Overview
+To try it out, add the `-Z sparse-registry` flag on `nightly-2022-06-19` or newer build
+of Cargo. For example, to update dependencies:
+
+```
+rustup update nightly
+cargo +nightly -Z sparse-registry update
+```
+
+The feature can also be enabled by setting the environment variable
+`CARGO_UNSTABLE_SPARSE_REGISTRY=true`. Setting this variable will have no effect on stable
```

Comment:
nit: I wonder if it's worth *only* mentioning this in the blog post as the way to opt-in. It just strikes me as the simpler of the two options.

Reply:
I think I like having both approaches. People familiar with nightly flags are used to them as `-Z` options.

I'm open to thoughts by others though.